app/_how-tos/create-a-key-set-with-a-jwk.md

```diff
@@ -36,6 +53,34 @@ cleanup:
       icon_url: /assets/icons/gateway.svg
 ---
 
-@todo
+## 1. Create a Key Set
+
+{% control_plane_request %}
+  url: /key-sets
+  method: POST
+  headers:
+      - 'Accept: application/json'
+      - 'Content-Type: application/json'
+      - 'Kong-Admin-Token: $KONG_ADMIN_TOKEN'
+  body:
+      name: my-key-set
+{% endcontrol_plane_request %}
+
+## 2. Create a Key
+
+Create a Key and use either the `set.id` or `set.name` parameter to add it to the Key Set.
 
-based on: https://docs.konghq.com/gateway/latest/reference/key-management/#create-a-key-using-the-jwk-format-and-associate-it-with-a-key-set
\ No newline at end of file
+{% control_plane_request %}
+  url: /keys
+  method: POST
+  headers:
+      - 'Accept: application/json'
+      - 'Content-Type: application/json'
+      - 'Kong-Admin-Token: $KONG_ADMIN_TOKEN'
+  body:
+      name: my-key
+      kid: my-key
+      set:
+        name: my-key-set
+      jwk: "{\"kty\":\"RSA\",\"e\":\"AQAB\",\"use\":\"enc\",\"kid\":\"my-key\",\"alg\":\"RSA1_5\",\"n\":\"n_03K8g2O_rarMBqBpbDKtRzrKede24g8UQ8Jc_x4-vsBnCFJw_xUcy-j4Ub9hYQZtyBZ5bWuEWC1crsorFgDbzoO1fF237XtCUCb0G6a8-3fbeSQZGwglK_vIy8-pHzZnOC2kgHp-rrNo9xZHnaOkrqqW4CI8izDuxboi_BlGqiNjKqGimj6fCPkiIEFlIrAtQCM9bUJDXv_iIs9blv9StqrfWnwxPIeIuoeruY_eC76twMweH5JHEAx_7BJdTdOXo9lrwmoUYwLAPp9w4E9Dc1lW1gQXh8aK4UUaJcsTjEztPtKsPHkQGSuP5WxM5uNH9Jo3-4wwuoA6BDxBS4sw\"}"
+{% endcontrol_plane_request %}
```

Comment:
How do we validate that this worked? Will retrieving the key set with a GET give us a list of keys in the set? If there's no way to tell, should at least add something like:

"You will see a 200 response with the key you just added and the Key Set ID in the response, which matches the Key Set you created earlier".